File: php/08series/series1.php
<?php
$sum = 0;
$hisob = 1;
while($hisob <= 10){
	$N = mt_rand(1, 100) / 4;
	$N = number_format($N, 2);
	echo "N".$hisob." = ".$N."<br>";
	$sum += $N;
	$hisob++;
}
echo "Ҷамъи ададҳо = ".$sum;
?>

/*
<?php
	$sum = 0;
	for ($i = 1; $i <= 10; ++$i) {
		$number = floatval(readline("number$i = "));
		$sum += $number;
	}
    echo sprintf('%.2f', $sum);
?>
*/

File: cpp/08series/Series1.cpp
#include <iostream>
using namespace std;

int main()
{
	//Task("Series1");
	double number, sum = 0;
	for (int i = 1; i <= 10; i++)
	{
		cin >> number;
		sum += number;
	}
	cout << sum;
	
	return 0;
}
